### DISCUSSION TITLE
Duplicates binary/debain

### DISCUSSION THREAD

**Discussion by tylerjw** [osrf/ros2_test_cases#1041] [General]
Here is an example. I posted this in discord too:

- https://github.com/osrf/ros2_test_cases/issues/755
- https://github.com/osrf/ros2_test_cases/issues/752

**Comment by tylerjw** [comment]
Closing, was clarified by @mjcarroll in the meeting.

**Reply by tfoote** [reply]
For others reference binaries refer to the tar/zip installation method. Versus the debians which technically are binary but install via debian/apt toolchain.
